branchout-yarn

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+set -e 
+test -n "${DEBUG}" || set -x 
+
+function runYarn() {
+    commands=("${@}")
+    HOME=${BRANCHOUT_HOME}/node 
+    YARN_CACHE_FOLDER=${BRANCHOUT_HOME}/node/cache yarn --use-yarnrc ${BRANCHOUT_HOME}/node/yarnrc ${@}
+}
+
+function yarnSettings() {
+    if test -z "${BRANCHOUT_NPM_REPOSITORY}"; then
+    readValue "your maven repository" "BRANCHOUT_NPM_REPOSITORY"
+    [[ ${BRANCHOUT_NPM_REPOSITORY} =~ ^https:// ]] || BRANCHOUT_NPM_REPOSITORY="https://${BRANCHOUT_NPM_REPOSITORY}"
+    "${BRANCHOUT_PATH}/branchout" set "BRANCHOUT_NPM_REPOSITORY" "${BRANCHOUT_NPM_REPOSITORY}"
+    fi
+    
+    readValue "npm repository username" "NPM_USER"
+    readSecret "npm repository secret" "NPM_PASS"
+    readValue "npm repository email (used for identification):" "NPM_EMAIL"
```

Comment:
Use ensureConfigValue, that will save/use value from branchout home/branchoutrc.

I think the email will be the same as used in the commit author email so saving is a good idea

Reply:
Good call! Updated and tested. Works well.